extensions/amp-access/0.1/amp-access-server.js

```diff
@@ -29,6 +29,13 @@ const TAG = 'amp-access-server';
 /** @const {number} */
 const AUTHORIZATION_TIMEOUT = 3000;
 
+function getSubtle(win) {
```

Comment:
So, looks like you don't need this method anymore. Right? It's resolved inline in the `getSubtleWithKey_` method.